python/ray/job_config.py

```diff
@@ -18,6 +18,8 @@ class JobConfig:
             ``runtime_env.py`` for detailed documentation).
         client_job: A boolean represent the source of the job.
         default_actor_lifetime: The default value of actor lifetime.
+        _py_driver_sys_path: A list of directories that
```

Comment:
Naming.. should we remove `_` prefix given that this is a public API?

Reply:
done